app/src/main/java/com/google/droidjump/Droid.java

```diff
@@ -20,34 +20,43 @@
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 
+/**
+ * Represents Droid character.
+ *
+ * @author maksme@google.com
+ */
 public class Droid extends DrawableElement {
 
+    // the variable is responsible for how much time ticks full animation of droid goes
```

Comment:
It seems like you have merged incorrectly, there are a lot of unnecessary changes. Please fix them first.